### PR TITLE
Wip mem corr

### PR DESCRIPTION
**Pull request by eladpe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3541/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1620/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3542/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1621/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1628/ for details.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/3548/ for details (Mellanox internal link).

File: src/ucs/stats/stats.c [hunk 1]
```diff
@@ -547,8 +622,12 @@ static void ucs_stats_clean_node_recurs(ucs_stats_node_t *node)
         ucs_warn("stats node "UCS_STATS_NODE_FMT" still has active children",
                  UCS_STATS_NODE_ARG(node));
     }
-
+//int end_less = 0;
     ucs_list_for_each_safe(child, tmp, &node->children[UCS_STATS_INACTIVE_CHILDREN], list) {
+//if (end_less++ > 20) {
```
Comment (shamisp) [inline src/ucs/stats/stats.c]: no c++ comments pls

Comment (eladpe) [inline src/ucs/stats/stats.c]: sorry, this commit is not for merge. I was debugging something that reconstruct only on CI

**Pull request closed without merging**